app/views/associate_ucr_summary.scala.html

```diff
@@ -30,7 +30,7 @@
         @components.back_link(consolidations.routes.AssociateUCRController.displayPage())
 
         @components.page_title(
-            title = Some(messages("associate.ucr.summary.title"))
```

Comment:
I don't think this is right.  The whole point of Title was that it forced the title to be constructed from the pattern "{0} - {1} - GOV.UK".    You can now by-pass this.